packages/@ourworldindata/utils/src/owidTypes.ts

```diff
@@ -826,6 +823,73 @@ export type EnrichedBlockCallout = {
     text: Span[][]
 } & EnrichedBlockWithParseErrors
 
+export type RawBlockTopicPageIntro = {
+    type: "topic-page-intro"
+    value: {
+        "download-button":
+            | {
+                  text: string
+                  url: string
+              }
+            | undefined
+        "related-topics":
+            | {
+                  text?: string
+                  url: string
+              }[]
+            | undefined
+        content: RawBlockText[]
+    }
+}
+
+export type EnrichedTopicPageIntroRelatedTopic = {
+    text?: string
+    url: string
+    type: "topic-page-intro-related-topic"
+}
+
+export type EnrichedTopicPageIntroDownloadButton = {
+    text: string
+    url: string
+    type: "topic-page-intro-download-button"
+}
+
+export type EnrichedBlockTopicPageIntro = {
+    type: "topic-page-intro"
+    downloadButton?: EnrichedTopicPageIntroDownloadButton
+    relatedTopics?: EnrichedTopicPageIntroRelatedTopic[]
+    content: EnrichedBlockText[]
+} & EnrichedBlockWithParseErrors
+
+export type RawBlockKeyInsightsSlide = {
+    title?: string
+    filename?: string
+    url?: string
+    content?: OwidRawGdocBlock[]
+}
+
+export type RawBlockKeyInsights = {
+    type: "key-insights"
+    value: {
+        heading?: string
+        insights?: RawBlockKeyInsightsSlide[]
+    }
+}
+
+export type EnrichedBlockKeyInsightsSlide = {
+    type: "key-insight-slide"
+    title: string
+    filename?: string
+    url?: string
```

Comment:
I went with `content` because authors also use prominent links in these things and `text` is (mostly, occasionally) used in contexts where it's either only plaintext or `Span`s.

But maybe I'm missing the point and your suggestion is more concerned with how to specify `key-insight-slide` as an item of narrative content more generally. 🙂

Reply:
Yeah the way you outline it also works. It's probably fine to extend it like that (i.e. keep content and add a possible .chart when we come to it)